app/content/react/guides/query-parameters.md

```diff
@@ -0,0 +1,124 @@
+## Query Parameters
+
+You can use QueryParameters to filter your layers & widgets at the backend side, using the concept of *parametrized queries* over your sources. You can check our related [deck.gl documentation]({{< ref "/deck-gl/reference#queryparameters-depends-on-provider-optional" >}} "About Us") for different providers.
```

Comment:
@Josmorsot This "About Us" looks wrong. Maybe a copy & paste. If we keep it, we should display something like "Query parameters in deck.gl"